core/src/main/java/hudson/model/EnvironmentContributingAction.java

```diff
@@ -46,23 +45,20 @@
  *
  * @author Kohsuke Kawaguchi
  * @since 1.318
- * @see AbstractBuild#getEnvironment(TaskListener)
+ * @see Run#getEnvironment(TaskListener)
  * @see BuildWrapper
  */
 public interface EnvironmentContributingAction extends Action {
     /**
-     * Called by {@link Run} or {@link AbstractBuild} to allow plugins to contribute environment variables.
+     * Called by {@link Run} to allow plugins to contribute environment variables.
      *
      * @param run
      *      The calling build. Never null.
-     * @param node
-     *      The node execute on. Can be {@code null} when the Run is not binded to the node,
-     *      e.g. in Pipeline outside the {@code node() step}
      * @param env
      *      Environment variables should be added to this map.
-     * @since TODO
+     * @since 2.76
      */
-    default void buildEnvVars(@Nonnull Run<?, ?> run, @Nonnull EnvVars env, @CheckForNull Node node) {
+    default void buildEnvironment(@Nonnull Run<?, ?> run, @Nonnull EnvVars env) {
```

Comment:
Changing name as well as signature; otherwise the overloads would be ambiguous without static typing, which is very error-prone.